apps/chatbot/pyproject.toml

```diff
@@ -37,7 +37,10 @@ llama-index-llms-gemini = "^0.3.4"
 google-generativeai = "^0.5.2"
 llama-index-embeddings-gemini = "^0.2.0"
 llama-index-llms-bedrock-converse = "^0.3.0"
+chromedriver-py = "^129.0.6668.91"
 llama-index-postprocessor-presidio = "^0.2.0"
+presidio-anonymizer = "^2.2.355"
+presidio-analyzer = "^2.2.355"
```

Comment:
remove `presidio-anonymizer` and `presidio-analyzer` and install `llama-index-postprocessor-presidio` as in the `pyproject.toml` in main. Check this out.

In the end, why `chromedriver-py`?